apps/web/lib/settings/member-table.tsx

```diff
@@ -59,23 +59,18 @@ export const MemberTable = ({ members }: { members: OT_Member[] }) => {
 
 		if (isPromotingToManager) {
 			// Add new manager
-			const updatedMemberIds = [...new Set([
-				...(activeTeamRef.current?.members || []).map((member: OT_Member) => member.employee.id),
-				employeeId
-			])];
+			const updatedManagerIds = [...new Set([...currentManagers, employeeId])];
 
 			return updateOrganizationTeam(activeTeamRef.current, {
 				...activeTeamRef.current,
-				memberIds: updatedMemberIds
+				managerIds: updatedManagerIds
```

Comment:
@Cedric921 so you adding that person into managers, but do you remove that same person from members when it happens!??

Reply:
Here's how it works: 
- when a member becomes a manager, we just add his id to the list of managers (managerIds), in the backend, we dont need to delete him from members  list 
- when we want to delete a manager role for a member, we just send the list of manager ids without the member's id. 

@evereq